src/compute/utils/file_utils.rs

```diff
@@ -228,55 +247,77 @@ mod tests {
 
     #[test]
     fn test_successful_download() {
-        let result = download_file(URL, PARENT_DIR, FILE_NAME);
+        let container = GenericImage::new("kennethreitz/httpbin", "latest")
+            .with_exposed_port(80.tcp())
+            .with_wait_for(WaitFor::message_on_stderr("Listening at"))
+            .with_network("bridge")
```

Comment:
Same remarks to avoid defining the network when using the default one and about duplications regarding container instance creation